docs/get-started/getting-started-azure-synapse-analytics.md

```diff
@@ -0,0 +1,19 @@
+---
+layout: page
+title: Azure Synapse Analytics
+nav_order: 5
+parent: Getting-Started
+---
+# Getting started with RAPIDS Accelerator on Azure Synapse Analytics
+ [Azure Synapse](https://docs.microsoft.com/en-us/azure/synapse-analytics/) is an analytics service that brings
+ together enterprise data warehousing and Big Data analytics.
+
+Synapse now offers the ability to create Apache Spark pools that use GPUs on the backend to run your Spark workloads on 
+GPUs for accelerated processing. This is called as 
+[GPU-accelerated Apache Spark pools](https://docs.microsoft.com/en-us/azure/synapse-analytics/spark/apache-spark-gpu-concept).
+Currently it ships with the RAPIDS Accelerator for Apache Spark version 21.10.
+
+Please follow below 2 quickstart guides:
```

Comment:
Please follow the below quickstart guides to learn how to create and use GPU pools in Azure Synapse: